roachtest: activerecord failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20734272?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20734272?buildTab=artifacts#/activerecord) on release-25.2.9-rc @ [f0bfb1cb00838ff45a508e4f1eba087e9835a674](https://github.com/cockroachdb/cockroach/commits/f0bfb1cb00838ff45a508e4f1eba087e9835a674):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.9-dev-f0bfb1cb00838ff45a508e4f1eba087e9835a674
Tests run against activerecord 8.0.1
8558 Total Tests Run
8555 tests passed
3 tests failed
67 tests skipped
39 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Encryption::EncryptableRecordApiTest#test_ciphertext_for_returns_the_ciphertext_of_a_new_value (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptableRecordApiTest#test_ciphertext_for_returns_the_ciphertext_of_a_value_when_the_record_is_new (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20734272/activerecord/1763102529280/1763105863105)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #157123 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #156991 roachtest: activerecord failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2 release-blocker]
- #156921 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.3.23-rc]
- #156441 roachtest: activerecord failed [test_migrate_revert_add_unique_constraint_with_invalid_option] [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.3 target-release-24.3.24]
- #155695 roachtest: activerecord failed [C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-25.3.3-rc]
- #152774 roachtest: activerecord failed [cached plan must not change result type] [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-3 T-sql-foundations branch-release-25.3]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56956

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -28,6 +28,8 @@ var activeRecordIgnoreList = blocklist{
 	`ActiveRecord::ConnectionAdapters::ConnectionPoolThreadTest#test_checkout_fairness_by_group`:                                                               "flaky",
 	`ActiveRecord::ConnectionAdapters::ConnectionPoolThreadTest#test_disconnect_and_clear_reloadable_connections_are_able_to_preempt_other_waiting_threads`:    "flaky",
 	`ActiveRecord::ConnectionAdapters::PostgreSQLAdapterTest#test_translate_no_connection_exception_to_not_established`:                                        "pg_terminate_backend not implemented",
+	`ActiveRecord::Encryption::EncryptableRecordApiTest#test_ciphertext_for_returns_the_ciphertext_of_a_new_value`:                                             "flaky",
+	`ActiveRecord::Encryption::EncryptableRecordApiTest#test_ciphertext_for_returns_the_ciphertext_of_a_value_when_the_record_is_new`:                          "flaky",
 	`ActiveRecord::Encryption::EncryptableRecordTest#test_by_default,_it's_case_sensitive`:                                                                     "flaky",
 	`ActiveRecord::Encryption::EncryptableRecordTest#test_forced_encoding_for_deterministic_attributes_will_replace_invalid_characters`:                        "flaky",
 	`ActiveRecord::Encryption::UniquenessValidationsTest#test_uniqueness_validations_work_when_using_old_encryption_schemes`:                                   "flaky",

